vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -211,6 +211,65 @@ public FormLayout() {
     public FormLayout(Component... components) {
         add(components);
     }
+    
+    /**
+     * Sets the colspan of the given component's element.
+     * You can directly add components with the wanted colspan with
+     * {@link #add(Component)}.
+     *
+     * @param component
+     *            the component to modify
+     *
+     * @param colspan
+     *            the desired colspan for the component
+     *
+     */
+    public void setColspan(Component component, int colspan) {
+    	String strColspan = "";
+    	if (colspan < 1) {
+    		strColspan = "1";
```

Comment:
>    	if (colspan < 1) {
>    		strColspan = "1";

This is a side effect and not mentioned in the javadocs. If you do this, you need to mention that what happens for values <1. Another option is to throw `IllegalArgumentException`

Reply:
Mentioned it in latest commit.